front_end/src/App.css

```diff
@@ -386,3 +386,89 @@ a:active {
   top: 23px;
   position: relative;
 }
+/* Payment Page */
+.row {
+  display: -ms-flexbox; /* IE10 */
+  display: flex;
+  -ms-flex-wrap: wrap; /* IE10 */
+  flex-wrap: wrap;
+  margin: 0 -16px;
+}
+
+.col-25 {
```

Comment:
please give your css classes more descriptive names. also, make sure not to use things like "container" as this is a bootstrap class. If you overwrite it in app.css, it will overwrite it for everybody that is using it.

call it something like "paymentPageContainer"